src/i18n/Plugins.ts

```diff
@@ -155,4 +155,16 @@ export default {
     zh: '为保证同步后插件正常运行，请在同步后重新启动插件',
     en: 'To ensure the plugin runs normally after synchronization, please restart the plugin after synchronization.',
   },
+  healthStatus: {
+    zh: '健康状态',
+    en: 'Health Status',
+  },
+  ok: {
```

Comment:
I'm not sure these status label is appropriate, please confirm

Reply:
LGTM